src/main/java/org/kohsuke/github/GHMyself.java

```diff
@@ -65,7 +47,7 @@ public List<String> getEmails() throws IOException {
      * @throws IOException
      *             the io exception
      */
-    public List<GHEmail> getEmails2() throws IOException {
+    public List<GHEmail> getEmails() throws IOException {
```

Comment:
TODO: something other than this. 
Need to mark this deprecated before making this change.
```suggestion
    @Deprecated
    public List<GHEmail> getEmails2() throws IOException {
```